File: src/test/resources/beta_1_1_1/Sample.pl
#!/usr/bin/perl

#If player is X, I'm the first player.
#If player is O, I'm the second player.
$player = <>;

#Read the board now. The board is a 3x3 array filled with X, O or _.
for ($i=0;$i<3;$i++) {
    $board[$i] = <>;
}

#Proceed with processing and print 2 integers separated by a single space.
#Example: print int(rand(3)), ' ', int(rand(3));

File: src/test/resources/beta_1_1_1/Sample.rb
#!/bin/ruby

#If player is X, I'm the first player.
#If player is O, I'm the second player.
player = gets;

#Read the board now. The board is a 3x3 array filled with X, O or _.
board = Array.new(3);

for i in Range.new(0, 2).to_a
    board[i] = gets;
end

#Proceed with processing and print 2 integers separated by a single space.
#Example: print Random.rand(3), ' ',  Random.rand(3)

